### PR TITLE
Ntg

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: conn-ntg/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+loginTimeout: 30000
+
+# Logout timeout in milliseconds.
+logoutTimeout: 5000
+
+# Idle timeout in milliseconds (if no message has been sent then client must sent Heartbeat message)
+heartbeatTimeout: 30000
+
+# Quantity of reconnect attempts after server connection drop
+reconnectAttempts: 10
```
Comment (Nikita-Smirnov-Exactpro) [inline conn-ntg/README.md]: Remove, this option isn't used in NTGClient

File: conn-ntg/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Connect Native Trading Gateway (NTG)
```
Comment (Nikita-Smirnov-Exactpro) [inline conn-ntg/README.md:1]: Please add information about the reconnectTimeout option with a default value 5000

File: conn-ntg/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+# User newPassword
+newPassword: ""
+
+# If true client sends heartbeat (response or timeout heartbeat).
+autosendHeartbeat: true
+
+# Version of the messages used in session.
+messageVersion: 1
+
+idleTimeout: 0
```
Comment (Nikita-Smirnov-Exactpro) [inline conn-ntg/README.md:51]: Please add description

File: conn-ntg/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+doLoginOnStart: true
```
Comment (Nikita-Smirnov-Exactpro) [inline conn-ntg/README.md:54]: Please add description

File: conn-ntg/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+# Connect timeout in milliseconds.
+connectTimeout: 30000
+
+# Login timeout in milliseconds.
+loginTimeout: 30000
+
+# Logout timeout in milliseconds.
+logoutTimeout: 5000
+
+# Idle timeout in milliseconds (if no message has been sent then client must sent Heartbeat message)
```
Comment (jean-michel-perli-sonsin) [inline conn-ntg/README.md]: Idle timeout in milliseconds (if no message has been sent then **the** client must sent **a** Heartbeat message)

File: conn-ntg/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+# User ID
+login: "user"
+
+# User password
+password: "password"
+
+# User newPassword
+newPassword: ""
+
+# If true client sends heartbeat (response or timeout heartbeat).
```
Comment (jean-michel-perli-sonsin) [inline conn-ntg/README.md]: If true, **the** client sends **a** heartbeat (response or timeout heartbeat).

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
Hello, I had add two comments (please see if you agree with them). The rest of the text itself looks very good, clear and easy for the users to understand.

File: conn-ntg/README.md (hunk 6 above)
Comment (IvanChadin) [reply conn-ntg/README.md]: done

File: conn-ntg/README.md (hunk 5 above)
Comment (IvanChadin) [reply conn-ntg/README.md]: done

File: conn-ntg/README.md (hunk 4 above)
Comment (IvanChadin) [reply conn-ntg/README.md:54]: done

File: conn-ntg/README.md (hunk 2 above)
Comment (IvanChadin) [reply conn-ntg/README.md:1]: done

File: conn-ntg/README.md (hunk 1 above)
Comment (IvanChadin) [reply conn-ntg/README.md]: done

File: conn-ntg/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+# User newPassword
+newPassword: ""
+
+# If true, the client sends a heartbeat (response or timeout heartbeat).
+autosendHeartbeat: true
+
+# Version of the messages used in session.
+messageVersion: 1
+
+# Incativity service timeout in seconds after which service will be shout down automatically. If 0 - do not shoutdown service.
```
Comment (jean-michel-perli-sonsin) [inline conn-ntg/README.md]: **Inactivity** service timeout in seconds after which service will be **shut** down automatically. If **the value is set to** 0 - do not **shutdown the** service.

File: conn-ntg/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+idleTimeout: 0
+
+# Send 'Logon' message at service startup
+doLoginOnStart: true
+
+# Reconnect will be executed with the specified timeout in milliseconds if service has lost the connection. 
+# If value is 0 (ZERO) service won't reconnect. For reconnect task work the following settings must be enabled: Do Login On Start, Autosend Heartbeat.
```
Comment (jean-michel-perli-sonsin) [inline conn-ntg/README.md]: If **the** value is **set to** 0 (ZERO) **, the** service won't reconnect. **In order for the reconnect task to work,** the following settings must be enabled: Do Login On Start, Autosend Heartbeat.

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
I just add two more comments, please review them when you can. Thanks!

File: conn-ntg/README.md (hunk 8 above)
Comment (IvanChadin) [reply conn-ntg/README.md]: done

File: conn-ntg/README.md (hunk 7 above)
Comment (IvanChadin) [reply conn-ntg/README.md]: done

**Review by IvanChadin — APPROVED** [review]

**Pull request merged**